modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc

```diff
@@ -0,0 +1,55 @@
+= Secure View
+
+When Collabora Online is properly setup and integrated into ownCloud Server (10.2 or above) Secure View functionality is available.
+Secure View is a mode where users can place limitations on files and folders that are shared. These limitations include:
+
+* No copying
+* No downloading
+* No editing
+* Watermarking with optional print and export
++
+NOTE: Administrators can configure the watermark text.
+* Printing and exporting to PDF with watermarks included
+
+IMPORTANT: Documents never leave the server when shared with Secure View.
+Collabora Online Server opens them and streams the files to the user's browser (much like a video stream).
+Consequently, there's no way to extract the original document from the browser.
+
+== Supported File Formats
+
+Secure View only supports a limited number of file formats; these are:
+
+* Microsoft Word (.docx)
+* Microsoft Excel (.xlsx)
+* Microsoft PowerPoint (.pptx)
+* OpenDocument Text Document (.odt)
+* OpenDocument Presentation Document (.odp)
+* OpenDocument Spreadsheet Document (.ods)
+* PDF
+
+If a folder shared with Secure View contains unsupported file types (e.g., JPG), they will not be accessible.
+
+== Prerequisites
+
+* ownCloud 10.2 or higher _Enterprise Edition_
+* Collabora Online set up and integrated
+
+NOTE: This functionality does not work with Public Links.
+
+== How to Enable Secure View
+
+To enable _Secure View_, navigate to menu:Settings[Admin > Additional (Admin) > Collabora Online].
+At the bottom of the Collabora Online section, enable btn:[Enable Secure View].
+
+Once enabled, default share permissions for all users can, optionally, be enabled. 
+Currently, these default share permissions are:
+
+* *can view with watermark*. 
```

Comment:
`can view with watermark` => `Secure View (with watermarks)`
`can print/export` => `can print / export PDF`

> Documents can be printed or exported through Collabora Online. If this option is disabled, documents can only be viewed with a watermark.

^ That's not correct. If the option is enabled, you share in Secure View mode, meaning all the properties above apply at once (no edit, no dl, no print/export, no copy, watermarks). If you disable that option can "can edit" is also disabled you have a regular "read-only" share.

> Documents can be viewed through Collabora Online, but will be watermarked.

`can print / export PDF` is an addition to `Secure View (with watermarks)`. It additionally allows recipients to physically print documents or export them to PDF including the watermarks from within Collabora Online.

Generally, please note that in this section only default values for user/group sharing are set, meaning admins can specify that all shares are "Secure View" shares by default and the user intentionally has to change that or vice versa.